.rubocop.yml

```diff
@@ -21,3 +20,107 @@ AllCops:
     - bin/bundle
     - bin/yarn
     - node_modules/**/*
+
+Layout/ClassStructure:
+  Enabled: true
+
+Layout/LineLength:
+  Max: 100
+
+Layout/SpaceBeforeFirstArg:
+  Exclude:
+    - app/views/api/**/**/*
+
+Lint/AmbiguousBlockAssociation:
+  Exclude:
+    - spec/**/*
+
+Lint/BinaryOperatorWithIdenticalOperands:
+  Enabled: false
+
+Lint/DeprecatedOpenSSLConstant:
+  Enabled: false
+
+Lint/RaiseException:
+  Enabled: false
```

Comment:
note for the future, we should double check these cops. Might be a good idea to turn them on (default)